src/platforms/native/platform.h

```diff
@@ -27,7 +27,6 @@
 #include "timing.h"
 #include "timing_stm32.h"
 
-#define TRACESWO_PROTOCOL 1U /* 1 = Manchester, 2 = NRZ / async */
```

Comment:
This change needs to be matched with the introduction of this define into the Makefile for native.